tests/test_namespaces.py

```diff
@@ -357,3 +357,31 @@ class TestModel(BaseTestModel, tag='model', ns='tst', nsmap={'tst': 'http://test
 
     actual_xml = actual_obj.to_xml()
     assert_xml_equal(actual_xml, xml1)
+
+
+def test_submodel_namespaces_default_namespace_inheritance():
+    class TestSubModel(BaseXmlModel, tag='submodel', ns='', nsmap={'': 'http://test2.org'}):
```

Comment:
note: Just want to point out with your suggested change, i have to set `ns=''` for it to work (which is perfectly fine for my use case)